### PR TITLE
Feature/single user lock

### PR DESCRIPTION
**Pull request by ankjevel**
So again, this contains ALL previous pull-requests. Don't know how else to do it...

### CODE REVIEW

File: lib/models/Job.js [hunk 1]
```diff
@@ -30,59 +32,61 @@ module.exports = class Job {
...
+    source = '',
+    sourceId = '',
+    startDate = null,
+    text = '',
+    title = '',
+    type = ''
   }) {
     this.id = id
-    this.type = Job.employment(type)
+    this.address = `${address}`
```
Comment (Jimjardland) [inline lib/models/Job.js]: No need to be template-string

File: lib/models/Job.js [hunk 2]
```diff
@@ -30,59 +32,61 @@ module.exports = class Job {
...
     this.company = typeof company === 'string' ? company : null
-    this.title = typeof title === 'string' ? title : null
-    this.preamble = striptags(preamble)
-    this.text = striptags(text)
+    this.contact = `${contact}`
```
Comment (Jimjardland) [inline lib/models/Job.js]: No need to be a template-string

File: lib/models/Job.js [hunk 3]
```diff
@@ -30,59 +32,61 @@ module.exports = class Job {
...
     this.createdAt = Date.parse(createdAt) > 0 ? createdAt : null
+    this.currency = typeof currency === 'string' ? currency : null
+    this.education = Job.ltreeArrayString(Job.ltreeArray(education))
+    this.endDate = Date.parse(endDate) > 0 ? endDate : null
+    this.entryBy = `${entryBy}`
```
Comment (Jimjardland) [inline lib/models/Job.js]: No need to be a template-string

**Comment by ankjevel** [comment]
The reason for using template strings is that I get type-hinting from my code-editor

**Pull request merged**